### ISSUE TITLE
how to run app

### ISSUE THREAD

**Issue by bobwatcherx** [StephanGocht/guiml#1]
you example code layouts . i get error like this
```
root@miop-HP-Convertible-x360-11-ab1XX:/home/miop/belajar/guiml/guiml/examples/layouts# python3 __main__.py 
Traceback (most recent call last):
  File "/home/miop/belajar/guiml/guiml/examples/layouts/__main__.py", line 10, in <module>
    from guiml.core import Window
ImportError: cannot import name 'Window' from 'guiml.core' (/home/miop/belajar/guiml/guiml/guiml/core.py)
root@miop-HP-Convertible-x360-11-ab1XX:/home/miop/belajar/guiml/guiml/examples/layouts# pip install guiml

```

**Comment by StephanGocht** [comment]
This repository is pretty much work in progress and not functional, yet. As the interfaces are extremely unstable the examples will break often.  I can let you know when it reaches a somewhat usable state.

I am currently prototyping with the todolists example, which should work by just calling `python3 examples/todolist` from the directory containing the requirenments.txt.

**Comment by StephanGocht** [comment]
The project has reached some kind of minimal viable product state and I just  released the first version (v0.1.0). The readme should contain sufficient instructions for installation and the documentation for use. So I am closing this issue.
